contracts/SSVNetwork.sol

```diff
@@ -627,18 +639,6 @@ contract SSVNetwork is UUPSUpgradeable, Ownable2StepUpgradeable, ISSVNetwork {
         if (operator.owner != msg.sender) revert CallerNotOwner();
     }
 
-    function _registerValidatorPublicKey(bytes calldata publicKey) private {
-        if (publicKey.length != 48) {
-            revert InvalidPublicKeyLength();
-        }
-
-        bytes32 hashedPk = keccak256(publicKey);
-        if (validatorPKs[hashedPk].owner != address(0)) {
-            revert ValidatorAlreadyExists();
-        }
-        validatorPKs[hashedPk] = Validator({owner: msg.sender, active: true});
-    }
```

Comment:
I dont think we were supposed to get rid of the whole function just some of it, i think we were supposed to keep:
        if (publicKey.length != 48) {
            revert InvalidPublicKeyLength();
        }
maybe move it to registerValidator and remove this function

Reply:
i see im wrong